spec/models/order_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  before(:each) do
+    @client = FactoryGirl.create(:client)
+    @client.id = 1
+    @client.save
+  end
+  context 'validation' do
+    it 'should be valid' do
+      expect(create(:order)).to be_valid
```

Comment:
@DmitrS it fails. Please add `, optional: true` to `belongs_to :driver` relation in `Order` model to make it pass.